universal-application-tool-0.0.1/app/views/admin/questions/QuestionPreview.java

```diff
@@ -43,7 +43,7 @@ public static ContainerTag renderQuestionPreview(QuestionType type) {
     ContainerTag contentContainer = div(innerContentContainer).withId("sample-question");
 
     return div(titleContainer, contentContainer)
-        .withClasses(
-            Styles.BG_GRADIENT_TO_BR, Styles.FROM_BLUE_200, Styles.TO_BLUE_400, Styles.W_3_5);
+        .withClasses(Styles.W_3_5)
+        .withStyle("background-color: #EEEDE8"); // TODO(NOW): Get this into tailwind config.
```

Comment:
Nit(blocking): mind finishing this before merging?

Reply:
I ended up rolling that change back out for now. We'll probably want to do a global styling change at some point, so we'll take care of it then.